tensorflow/cc/BUILD

```diff
@@ -716,3 +717,26 @@ tf_cc_test(
         "//tensorflow/core:testlib",
     ],
 )
+
+transitive_hdrs(
+    name = "headers",
+    deps = [
+        ":cc_ops",
+        ":client_session",
+        ":gradients",
+        ":gradient_checker",
+        ":scope",
+        ":ops",
+        ":queue_runner",
+        ":coordinator",
+        ":remote_fused_graph_ops",
+        "//tensorflow/cc/profiler:profiler",
+        "//tensorflow/cc/saved_model:constants",
+        "//tensorflow/cc/saved_model:signature_constants",
+        "//tensorflow/cc/saved_model:tag_constants",
+        "//tensorflow/cc/saved_model:loader",
+        "//tensorflow/cc/saved_model:reader",
+        "//tensorflow/cc/tools:freeze_saved_model",
+    ],
+    visibility = ["//visibility:private"],
```

Comment:
I think private means it is only accessible from this BUILD file. You will need to grant wider access to this.
Maybe tensorflow:__subpackages__ (not 100% sure on the syntax)

Reply:
done. pretty sure its `visibility = ["//tensorflow:__subpackages__"],`